modules/virt-configuring-cluster-real-time.adoc

```diff
@@ -0,0 +1,140 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-cluster-realtime-workloads.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-configuring-cluster-real-time_{context}"]
+= Configuring a cluster for real-time workloads
+
+You can configure an {product-title} cluster to run real-time workloads.
+
+.Prerequisites
+* You have access to the cluster as a user with `cluster-admin` permissions.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the Node Tuning Operator.
+
+.Procedure
+
+. Label a subset of the compute nodes with a custom role, for example, `worker-realtime`:
++
+[source,terminal]
+----
+$ oc label node <node_name> node-role.kubernetes.io/worker-realtime=""
+----
++
+[NOTE]
+====
+You must use the default `master` role for {sno} and compact clusters.
+====
+
+. Create a new `MachineConfigPool` manifest that contains the `worker-realtime` label in the `spec.machineConfigSelector` object:
++
+.Example `MachineConfigPool` manifest
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  name: worker-realtime
+  labels:
+    machineconfiguration.openshift.io/role: worker-realtime
+spec:
+  machineConfigSelector:
+    matchExpressions:
+      - key: machineconfiguration.openshift.io/role
+        operator: In
+        values:
+          - worker
+          - worker-realtime
+  nodeSelector:
+    matchLabels:
+      node-role.kubernetes.io/worker-realtime: "" 
+----
++
+[NOTE]
+====
+You do not need to create a new `MachineConfigPool` manifest for {sno} and compact clusters.
+====
+
+. If you created a new `MachineConfigPool` manifest in step 2, apply it to the cluster by using the following command:
++
+[source,terminal]
+----
+$ oc apply -f <real_time_mcp>.yaml
+----
+
+. Create a `PerformanceProfile` manifest that applies to the labeled nodes and the machine config pool that you created in the previous steps:
++
+.Example `PerformanceProfile` manifest
+[source,yaml]
+----
+apiVersion: performance.openshift.io/v2
+kind: PerformanceProfile
+metadata:
+  name: profile-1
+spec:
+  cpu:
+    isolated: 4-39,44-79
+    reserved: 0-3,40-43
+  globallyDisableIrqLoadBalancing: true
+  hugepages:
+    defaultHugepagesSize: 1G
+    pages:
+    - count: 8
+      size: 1G
+  realTimeKernel:
+    enabled: true
+  workloadHints:
+    highPowerConsumption: true
+    realTime: true
+  nodeSelector:
+    node-role.kubernetes.io/worker-realtime: ""
+  numa:
+    topologyPolicy: single-numa-node
+----
+
+. Apply the `PerformanceProfile` manifest:
++
+[source,terminal]
+----
+$ oc apply -f <real_time_pp>.yaml
+----
++
+[NOTE]
+====
+The compute nodes automatically reboot twice after you apply the `MachineConfigPool` and `PerformanceProfile` manifests. This process might take a few minutes.
```

Comment:
It takes much more than a few minutes, so this might be confusing, and lead the user to assume that if the process is not completed after 20-30 minutes (and it probably won't) then there's something wrong.
I would rephrase it to something that reflects the fact the even more than a 1 hour wait if possible.

Reply:
Changed to "This process might take a long time".